openc3-cosmos-cmd-tlm-api/app/controllers/storage_controller.rb

```diff
@@ -25,6 +25,7 @@
 
 class StorageController < ApplicationController
   def buckets
+    return unless authorization('system')
```

Comment:
This is a good check but doesn't everyone have `system`?

Reply:
Yes, but at least not it requires you to be authenicated.  Before even not logged in could get it.